dbt/adapters/sqlserver/connections.py

```diff
@@ -157,7 +157,18 @@ def open(cls, connection):
                 con_str.append(f"Encrypt={credentials.encrypt}")
 
             con_str_concat = ';'.join(con_str)
-            logger.debug(f'Using connection string: {con_str_concat}')
+            
+            index = []
+            for i, elem in enumerate(con_str):
+                if 'pwd=' in elem.lower():
+                    index.append(i)
+                    
+            if len(index) !=0 :
+                con_str[index[0]]="PWD=***"
+
+            con_str_display = ';'.join(con_str)
+            
+            logger.debug(f'Using connection string: {con_str_display}')
```

Comment:
Replacing the actual creds with *** in `dbt logs` as users might be using this log file for analysis like what all auth types are being used and how frequently.
So rather than commenting the part, replaced the creds with ***
![image](https://user-images.githubusercontent.com/67547199/94122632-6b682100-fe70-11ea-86a8-edb1e1445940.png)